Core/Steps/SubSteps/AddFixVersionsForNewReleaseBranchSubStep.cs

```diff
@@ -0,0 +1,46 @@
+using Remotion.ReleaseProcessAutomation.Extensions;
+using Remotion.ReleaseProcessAutomation.Jira;
+using Remotion.ReleaseProcessAutomation.Jira.ServiceFacadeImplementations;
+using Remotion.ReleaseProcessAutomation.ReadInput;
+using Remotion.ReleaseProcessAutomation.SemanticVersioning;
+using Spectre.Console;
+
+namespace Remotion.ReleaseProcessAutomation.Steps.SubSteps;
+
+public interface IAddFixVersionsForNewReleaseBranchSubStep
+{
+  public void Execute (SemanticVersion currentVersion, SemanticVersion nextJiraVersion);
+}
+
+public class AddFixVersionsForNewReleaseBranchSubStep : AddMoveCreateFixVersionsSubStepBase, IAddFixVersionsForNewReleaseBranchSubStep
+{
+  public AddFixVersionsForNewReleaseBranchSubStep (
+      IAnsiConsole console,
+      IInputReader inputReader,
+      IJiraIssueService jiraIssueService,
+      IJiraVersionCreator jiraVersionCreator)
+      : base(console, inputReader, jiraIssueService, jiraVersionCreator)
+  {
+  }
+
+  public void Execute (SemanticVersion currentVersion, SemanticVersion nextJiraVersion)
+  {
+    var issues = JiraIssueService.FindAllNonClosedIssues(currentVersion.ToString());
+    Console.WriteLine($"These are some of the issues that are open for the current version '{currentVersion}':");
```

Comment:
Unfortunate that this looks exactly like a "normal" `Console.WriteLine()` call so there is no easy guard against not using the `IAnsiConsole`

Reply:
We might want to change all console fields to some better name, but I think that should be an extra ticket?

Reply:
yes, extra ticket. And I agree with @a-ctor  when reviewing without highlighting. I have no good naming solutions, fortunately, that's out of scope :)